src/extension/android/logCatMonitor.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as vscode from "vscode";
+import * as Q from "q";
+
+import {ISpawnResult} from "../../common/node/childProcess";
+import {CommandExecutor} from "../../common/commandExecutor";
+
+/* This class will print the LogCat messages to an Output Channel. The configuration for logcat can be cutomized in
+   the .vscode/launch.json file by defining a setting named logCatArguments for the configuration being used. The
+   setting accepts values as:
+      1. an array: ["*:S", "ReactNative:V", "ReactNativeJS:V"]
+      2. a string: "*:S ReactNative:V ReactNativeJS:V"
+   Type `adb logcat --help` to see the parameters and usage of logcat
+*/
+export class LogCatMonitor implements vscode.Disposable {
+    private static DEFAULT_PARAMETERS = ["*:S", "ReactNative:V", "ReactNativeJS:V"];
+
+    private _commandExecutor: CommandExecutor;
+
+    private _outputChannel = vscode.window.createOutputChannel("LogCat");
```

Comment:
In my tests I always saw a single output channel, so I'm guessing it gets reused.